nix/overlay.nix

```diff
@@ -4,9 +4,9 @@ final: previous:
     haskellOverlay = final.callPackage ./haskell-overlay.nix {};
 
     haskellPackages =
-      previous.haskell.packages.ghc94.extend final.curl_channable.haskellOverlay;
+      previous.haskell.packages.ghc96.extend final.curl_channable.haskellOverlay;
```

Comment:
We already did this before, but should this actually override the GHC version? With this setup you can't actually use the overlay as an overlay, because it will wipe out previous overlays and replace the GHC version. This is a problem that exists in our other repos too so it's not something we need to solve here, but it's probably good to at least be a ware of it.

Reply:
fair point. now that the package is exposed directly in the flake, we at least do have other ways for users to acquire the package that does not force them to use the overlay